index.js

```diff
@@ -130,7 +144,13 @@ module.exports = {
         } else {
             var config = params.config;
             if (!config) {
-                config = {params: {}, runMaster: true, runWorker: true, version: params.version};
+                var pckgJson;
+                try {
+                    pckgJson = parent.require('./package.json');
```

Comment:
Please remove. We need to discourage this pattern. Using parent, have been causing troubles in several cases like automated tests, etc. Version should be passed to ut-run from the place it is invoked.

Reply:
done: https://github.com/softwaregroup-bg/ut-run/pull/29/commits/608081d554c7fd4e3f9c58851201d675337adbda

now if a version has to be passed that has to be done explicitly as a parameter to ut-run.run